### PR TITLE
Api comments

### LINKED ISSUE #8: [4pt]  - Create feature: add new comment - Student B
Implement the "add new comment" form on the Comments pop up according to the wireframe:

Screenshot 2021-06-27 at 22 32 36

When the user clicks on the "Comment" button, the data is recorded in the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen is updated.

### PR DESCRIPTION
**Pull request by taldr27**
## Post and get comments from the involvement API 
In this milestone we:
- Use the get and post involvement API to have the new comments in the Popup.
- Refactor functions to have the new comments.
- Added styling to the add comment section.
- Added descriptive comments to popup creation.

### CODE REVIEW

**Comment by Haywayaheadshot** [comment]
# STATUS: Approved :ok_hand::clap:

Hi, @taldr27  good job implementing the comments on the pop ups!

Congratulations! 🎉



## Highlights :1st_place_medal:

:heavy_check_mark: Well-structured code in your JavaScript file.
:heavy_check_mark:  Good work commenting on each part of your code, it's easy to read and make a review.
:heavy_check_mark: Great HTML & CSS best practices.

Cheers and Happy coding! :clap: :clap: :clap:

**Review by Haywayaheadshot — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -33,4 +33,15 @@ window.addEventListener('DOMContentLoaded', pop);
...
+// Create new ID
+// const add = async () => {
+//   await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/', {
+//     method: 'POST',
+//     headers: {
+//       'Content-type': 'application/json; charset=UTF-8',
+//     },
+//   })
+//     .then((response) => console.log(response.json()));
+// };
```
Comment (Haywayaheadshot) [inline src/index.js:46]: Please delete unused codes. The code does not follow best practices as it has a redundant function declaration and unnecessary comments. Also, console.log(response.status) does not handle errors properly and the console.log(response.json()) is not awaited.

File: src/modules/add-comments.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  //   .then((response) => console.log('comment' + response.status));
+  //   e.target.reset();
+  //   document.querySelector('#input-name').value = '';
+  //   document.querySelector('#input-score').value = '';
+  //   console.log(idBtn);
+  here(idBtn);
+};
```
Comment (Haywayaheadshot) [inline src/modules/add-comments.js:20]: The code follows some good practices for JavaScript, but could use a few improvements:

`import here from './comments.js'`: It's good to import dependencies in a module, but the name here is not a descriptive name for the imported module. Consider renaming it to something more descriptive.

`await fetch`(: It's good to use the await operator when making an asynchronous HTTP request. This makes it easier to read and reason about the code.

.`then((response) => console.log('comment' + response.status)`): It's good to log the response status to see if the request was successful or not, but it is not used.

Error handling: The code doesn't handle any errors that might occur during the fetch. Consider using a try-catch block to catch any errors that might occur and log or display an error message to the user.

Also, Please delete unused codes. The code does not follow best practices as it has a redundant function declaration and unnecessary comments.

Overall, the code is on the right track, but a few changes can make it even better.

File: src/modules/popup.js [hunk 3]
```diff
@@ -2,21 +2,25 @@ const popupContainer = document.querySelector('.popup-container');
...
+  // Create div to append the popup items
   const containerPopUp = document.createElement('div');
   containerPopUp.className = 'containerPopUp';
   myPop.appendChild(containerPopUp);
 
+  // Create the image for each show, we set the image in the change.js file
   const imgSerie = document.createElement('img');
   imgSerie.id = 'imgSerie';
   imgSerie.className = 'image-popup';
   // imgSerie.src = completeResponse[i].image.medium;
```
Comment (Haywayaheadshot) [inline src/modules/popup.js:20]: Please delete unused codes. The code does not follow best practices as it has a redundant function declaration and unnecessary comments.

File: src/modules/popup.js [hunk 4]
```diff
@@ -26,41 +30,81 @@ const pop = async () => {
...
+  // Create h2 to append the title
   const titleSerie = document.createElement('h2');
   titleSerie.id = 'titleSerie';
   // titleSerie.innerHTML = completeResponse[i].name;
   description.appendChild(titleSerie);
 
+  // Create h2 to append the language of the show
   const languageSerie = document.createElement('h2');
   languageSerie.id = 'languageSerie';
   // languageSerie.innerHTML = completeResponse[i].language;
```
Comment (Haywayaheadshot) [inline src/modules/popup.js:47]: Please delete unused codes. The code does not follow best practices as it has a redundant function declaration and unnecessary comments.

**Review by Haywayaheadshot — COMMENTED** [review]
Hi @taldr27 

Great work so far 👏 
You've done well implementing the requirements of the project, but there are some few changes I would like to suggest. Please check the comments left below. I will be available on slack @Haywayaheadshot if you need any clarifications.
image


 ## Highlights
:+1: Readme file
:+1: no linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by taldr27** [comment]
Thanks for your review @Haywayaheadshot, I'm going to start working on your requested changes and submit the project for a new review now.

**Pull request merged**